### ISSUE TITLE
@babel/preset-env

### ISSUE THREAD

**Issue by thernstig** [babel/website#2497]
https://babeljs.io/docs/en/babel-preset-env#how-does-it-work mentions:

> We leverage these data sources to maintain [mappings of which version](https://github.com/babel/babel/blob/master/packages/babel-compat-data/data/plugins.json) of our supported target environments gained support of a JavaScript syntax or browser feature, as well as a mapping of those syntaxes and features to Babel transform plugins and core-js polyfills.

If I understood it correctly, that text is somewhat misleading. The [mappings of which version](https://github.com/babel/babel/blob/master/packages/babel-compat-data/data/plugins.json) part only lists plugins that will _transform_ the code. It is written like it is a complete list of of all Javascript and syntax features to certain browser versions.

Would it not be more clear to somehow re-write the text so that the link to `plugins.json` is just used for _transpilation_ i.e. _Babel transform plugins_ (which I think it is?) and then for the polyfill part with core-js separately (with another link if there is one for those  mappings?)

**Comment by nicolo-ribaudo** [comment]
You are right, data for the different polyfills are maintained in the polyfill itself or in the integration plugins:
- [core-js@3](https://www.npmjs.com/package/core-js-compat)
- [core-js@2](https://github.com/babel/babel/blob/main/packages/babel-compat-data/data/corejs2-built-ins.json) (this will be removed in Babel 8)
- [es-shims](https://github.com/babel/babel-polyfills/blob/main/packages/babel-plugin-polyfill-es-shims/data/polyfills.json)
